### PR TITLE
cloud: refine the instructions of AWS S3 access configurations

### PR DESCRIPTION
**Pull request by qiancai**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [ ] v6.2 (TiDB 6.2 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tidb-cloud/migrate-from-amazon-s3-or-gcs.md [hunk 1]
```diff
@@ -13,150 +13,150 @@ This document describes how to use Amazon Simple Storage Service (Amazon S3) or
...
 ### Step 1. Create an Amazon S3 bucket and prepare source data files
 
-1. Create an Amazon S3 bucket in your corporate-owned AWS account. 
+1. Create an Amazon S3 bucket in your corporate-owned AWS account.
 
     For more information, see [Creating a bucket](https://docs.aws.amazon.com/AmazonS3/latest/userguide/create-bucket-overview.html) in the AWS User Guide.
 
+    > **Note:**
+    >
+    > To minimize egress charges and latency, locate your Amazon S3 bucket and TiDB Cloud database cluster in the same region.
```
Comment (hfxsd) [inline tidb-cloud/migrate-from-amazon-s3-or-gcs.md:30]:
```suggestion
    > To minimize egress charges and latency, create your Amazon S3 bucket and TiDB Cloud database cluster in the same region.
```

File: tidb-cloud/migrate-from-amazon-s3-or-gcs.md [hunk 2]
```diff
@@ -13,150 +13,150 @@ This document describes how to use Amazon Simple Storage Service (Amazon S3) or
...
 
     For more information, see [Install TiUP](/tidb-cloud/migrate-data-into-tidb.md#step-1-install-tiup) and [Export data from MySQL compatible databases](/tidb-cloud/migrate-data-into-tidb.md#step-2-export-data-from-mysql-compatible-databases).
 
-> **Note:**
-> 
-> - Ensure that your source data can be copied to a file format supported by TiDB Cloud. The supported formats include CSV, Dumpling, and Aurora Backup Snapshot. If your source files are in the CSV format, you need to follow [the naming convention supported by TiDB](https://docs.pingcap.com/tidb/stable/migrate-from-csv-using-tidb-lightning#file-name). 
-> - Where possible and applicable, it is recommended that you split a large source file into smaller files of maximum size 256 MB because it allows TiDB Cloud to read files in parallel across threads, thereby resulting in potentially enhanced import performance.
-
-### Step 2. Configure Amazon S3 access 
+3. If your source data is in local files, you can upload the files to the Amazon S3 bucket using either AWS Web console or AWS CLI.
```
Comment (hfxsd) [inline tidb-cloud/migrate-from-amazon-s3-or-gcs.md:36]:
```suggestion
3. If your source data is in local files, you can upload the files to the Amazon S3 bucket using either the Amazon S3 Console or the AWS CLI.
```

File: tidb-cloud/migrate-from-amazon-s3-or-gcs.md [hunk 3]
```diff
@@ -13,150 +13,150 @@ This document describes how to use Amazon Simple Storage Service (Amazon S3) or
...
 
-To allow TiDB cloud to access the source data in your Amazon S3 bucket, you need to configure the Amazon S3 for each TiDB Cloud as a service on the AWS project and Amazon S3 bucket pair. Once the configuration is done for one cluster in a project, all database clusters in that project can access the Amazon S3 bucket.
+    - To upload files using the AWS Web console, see [Uploading objects](https://docs.aws.amazon.com/AmazonS3/latest/userguide/upload-objects.html) in the AWS User Guide.
```
Comment (hfxsd) [inline tidb-cloud/migrate-from-amazon-s3-or-gcs.md:38]:
```suggestion
    - To upload files using the Amazon S3 Console, see [Uploading objects](https://docs.aws.amazon.com/AmazonS3/latest/userguide/upload-objects.html) in the AWS User Guide.
```

**Review by hfxsd — APPROVED** [review]
Rest LGTM

**Review by sunzhaoyang — APPROVED** [review]
LGTM

File: tidb-cloud/troubleshoot-import-access-denied-error.md [hunk 4]
```diff
@@ -5,15 +5,17 @@ summary: Learn how to troubleshoot access denied errors when importing data from
...
 After clicking **Validate** on the **Data Import Task** page on the TiDB Cloud console, if you see an error message with the keyword `AccessDenied`, an access denied error has occurred.
 
 To troubleshoot the access denied errors, perform the following checks in the AWS Management Console.
 
 ## Check the policy of the IAM role
 
-In the AWS Management Console, go to **IAM** > **Access Management** > **Roles**, find the role you have created for the target TiDB cluster, and check the **Permission policies**. Check each policy and make sure that the `Resource` fields in each policy are correctly configured. The following is a sample policy:
+In the AWS Management Console, go to **IAM** > **Access Management** > **Roles**, find the role you have created for the target TiDB cluster, and check the **Permission policies**. Check each policy and make sure that the `Resource` fields in each policy are correctly configured.
+
+The following is an example policy.
```
Comment (qiancai) [inline tidb-cloud/troubleshoot-import-access-denied-error.md:18]:
```suggestion
The following is a sample policy.
```

**Pull request merged**